src/main/java/com/dj/core/serializer/ModelWrapper.java

```diff
@@ -11,11 +12,14 @@
 
     private final List<Neuron> inputLayer;
     private final List<Neuron> outputLayer;
+    private final Context context;
 
     private ModelWrapper(final List<Neuron> inputLayer,
-                        final List<Neuron> outputLayer) {
+                         final List<Neuron> outputLayer,
+                         final Context context) {
```

Comment:
@b0noI Usually, context is the 1st param.

Reply:
you are right, fixed, thank you!